scripts/bfb-multi-install

```diff
@@ -0,0 +1,302 @@
+#!/bin/bash
+
+###############################################################################
+#
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES, ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of NVIDIA CORPORATION &
+# AFFILIATES (the "Company") and all right, title, and interest in and to the
+# software product, including all associated intellectual property rights, are
+# and shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+#
```

Comment:
I hope we're not planning to add proprietary code into this otherwise open source package.